input-plugins/input-plugin-reference/index.js

```diff
@@ -31,9 +31,10 @@ module.exports = {
       options: [],
       settings: {
         empty: false,
```

Comment:
I think a story for that please! I think there is more expansion on this plugin than just that, so let that part stay OOScope for now...but not my `remove empty` request...that's the good kind of OOScope - ie very smartypants.